dali/test/python/test_operator_brightness_contrast.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import nvidia.dali as dali
+import nvidia.dali.fn as fn
+from nvidia.dali import pipeline_def
+import numpy as np
+from test_utils import compare_pipelines
+from test_utils import RandomDataIterator
+
+
+def bricon_ref(input, brightness, brightness_shift, contrast, contrast_center):
+  return brightness_shift + brightness * (contrast_center + contrast * (input - contrast_center))
+
+def ref_operator(contrast_center):
+  return lambda input, brightness, brightness_shift, contrast: \
+                bricon_ref(input, brightness, brightness_shift, contrast, contrast_center)
+
+def params():
+  return fn.random.uniform(range=[0.5, 2.0], seed=123), \
+         fn.random.uniform(range=[0.5, 2.0], seed=123), \
+         fn.random.uniform(range=[0.4, 0.6], seed=123)
+
+
+@pipeline_def(num_threads=12, device_id=0, seed=1234)
```

Comment:
that's quite a high number of threads. Maybe 4?

Reply:
done